docs-chef-io/content/server/upgrades.md

```diff
@@ -76,6 +76,23 @@ Set the `postgresql['pg_upgrade_timeout']` attribute in [chef-server.rb]({{< rel
        vacuumdb: vacuuming database "template1"
     ```
 
+1. Run `ANALYZE` on the PostgreSQL database.This process gathers statistics for the query planner to create the most efficient query execution paths. Accurate statistics will help the planner to choose the most appropriate query plan, and thereby improve the speed of query processing.
+    ```bash
+       sudo su - opscode-pgsql
+       /opt/opscode/embedded/bin/vacuumdb --all --analyze-only
+    ```
+
+   You should then see output like:
+   
+    ```bash
+       vacuumdb: vacuuming database "bifrost"
+       vacuumdb: vacuuming database "oc_id"
+       vacuumdb: vacuuming database "opscode-pgsql"
+       vacuumdb: vacuuming database "opscode_chef"
+       vacuumdb: vacuuming database "postgres"
+       vacuumdb: vacuuming database "template1"
```

Comment:
Interesting - the changing flags do not seem to impact the output? It still looks like the db is being vacuumed.

Reply:
`/opt/opscode/embedded/bin/vacuumdb` is tool which accepts options like `--analyze-only` or `--full`. So the output is same if we use vacuumdb script.Both commands can be run on the psql console but needs to give the database names.